GliaWidgets/CoreSDKClient/CoreSDKClient.Mock.swift

```diff
@@ -1,3 +1,4 @@
+import Foundation
```

Comment:
let's make a newline after the import section.

Reply:
Fixed 7c23e635f77daa3dacb46bcaaab2e852d2ed04e8